boxsdk/object/item.py

```diff
@@ -5,11 +5,56 @@
 import json
 
 from .base_object import BaseObject
+from boxsdk.config import API
 
 
 class Item(BaseObject):
     """Box API endpoint for interacting with files and folders."""
 
+    @classmethod
```

Comment:
Uploading a file is a method of a File object, so it's tied to a file. One option would be making one preflight method to File for new file uploads, and adding one different preflight method to File for updating existing files on Box. In that way, nothing needs to be a class/static method. But we have two preflight methods. But that might be a good idea actually. Since code duplication can be addressed by putting code needs to be shared in Item. I think I like this idea more!